src/HttpSession.h

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2021, Group5 Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file. See the AUTHORS file for names of contributors.
+
+#pragma once
+#include<iostream>
+#include<string>
+// SocketSession. 
+#include "SocketSession.h"
+#include "HttpUtility.h"
+class SocketSession;
+
+/**	@breif Handle the incoming http request. */
+class HttpSession{
+private:
+	/**	@breif socket id.*/
+	int socket_;
+public:
+
+	HttpSession(const int &socket);
+	HttpSession(const HttpSession &) = delete;
+	/**@breif Run socketSession after initial.*/
+	void run();
+	/** @breif Receive stream package.
+	 *	@param msg Stream package's content.
+	 */
+	void receiveMessage(const std::string &msg);
```

Comment:
Who calls this? If it's only used internally, then please make it private.

Reply:
Done